FLEDGE_Key_Value_Server_API.md

```diff
@@ -65,6 +65,12 @@ HTTPS is used to transport data. The method is `POST`.
 
 The HTTP POST body is encrypted.
 
+#### Cleartext headers
+
+Requests contain a cleartext HTTP `Content-Type` header with value `message/ad-auction-trusted-signals-request`.
```

Comment:
Likewise, it may be clearer to reword this sentence to something like:
Requests specify Content Types via these headers:
```
Content-Type: message/ad-auction-trusted-signals-request
Accept: message/ad-auction-trusted-signals-response
```